tools/epicseg/epicseg_normalizecounts.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="epicseg_normalizecounts" name="EpiCSeg - Normalize counts" version="0.1.0">
+	    <macros>
+		<import>macros.xml</import>
+	    </macros>
+	    <requirements>
+		<requirement type="package" version="1.0">epicseg</requirement>
+	    </requirements>
+	    <command detect_errors="exit_code"><![CDATA[
+
+		    #for $dataset in $datasets#
+              ln -s ${dataset.counts} `basename ${dataset.counts}` &&
+            #end for#
+
+            Rscript '${__tool_directory__}/epicseg.R' normalizecounts
+                --nthreads $nthreads
+
+            #for $dataset in $datasets#
+                --counts `basename ${dataset.counts}`
+            #end for# &&
+
+            rename "s/.dat/.txt/" *_norm.dat &&
```

Comment:
You don't need to rename files. In Galaxy files do not have names and file endings.